docs/cas-server-documentation/installation/Cassandra-ticket-registry.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: default
+title: CAS - Cassandra Ticket Registry
+---
+
+# Cassandra Ticket Registry
+Cassandra integration is enabled by including the following dependency in the WAR overlay:
+
+```xml
+<dependency>
+    <groupId>org.apereo.cas</groupId>
+    <artifactId>cas-server-support-cassandra-ticket-registry</artifactId>
+    <version>${cas.version}</version>
+</dependency>
+```
+
+## Configuration Considerations
+
+There are four core configuration concerns with Cassandra:
+
+1. Keyspace
+2. Ticket cleaning
+3. Object serialization mechanism
+4. Multi datacenter replication
+
+### Keyspace
+We suggest the following Keysapace definition:
+
+- String serialized ticket:
+```cql
```

Comment:
It's difficult as it's on the keyspace where you can declare the replication and many other configurations.Of course we could find a way of doing it, but is it worthy? In any case, I would do it in a future version, "now you can copy paste this"